deployment/helm/kuksa-cloud/Chart.yaml

```diff
@@ -0,0 +1,48 @@
+######################################################################
+# Copyright (c) 2020 Bosch.IO GmbH [and others]
+#
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+########################################################################
+---
+apiVersion: v2
```

Comment:
You write you use v3 but this looks like v2?

Reply:
I developed against Helm 3, which uses apiVersion: v2 (according to https://v3.helm.sh/docs/topics/charts/#the-apiversion-field)

Reply:
understood. Thanks!